doc/advanced_configuration.rst

```diff
@@ -142,6 +143,29 @@ If you so desire you can implement your own sorting key. It will be
 provided the relative paths to `conf.py` of each sub gallery folder.
 
 
+.. _within_gallery_order:
+
+Sorting gallery examples
+========================
+
+Within a given gallery (sub)section, the example files by default are ordered
+by the amount of code, i.e. :class:`sphinx_gallery.sorting.AmountOfCodeSortKey`
+as::
+
+    from sphinx_gallery.sorting import AmountOfCodeSortKey
```

Comment:
Why not `NumberOfLines` sortkey? I know that's kind of a bikeshed but I think it's more explicit